framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -258,7 +258,12 @@ def index(self):
             burst_config = BurstConfiguration(project.id)
             common.add2session(common.KEY_BURST_CONFIG, burst_config)
 
-        is_simulator_load = common.get_from_session(common.KEY_IS_SIMULATOR_LOAD) or False
+        if burst_config.start_time is not None:
+            is_simulator_load = True
+            common.add2session(common.KEY_IS_SIMULATOR_LOAD, is_simulator_load)
+        else:
+            is_simulator_load = common.get_from_session(common.KEY_IS_SIMULATOR_LOAD) or False
+
         is_simulator_copy = common.get_from_session(common.KEY_IS_SIMULATOR_COPY) or False
```

Comment:
Shouldn't this be inside `else` too? What happens when you copy a simulation, launch it and then refresh the page?

Reply:
What I said in another comment:
"I have also found another problem which could have something to do with my previous paragraph and this is about copying/branching a simulation (I think we may have discussed this already, I don't remember exactly). For example, let's say I launched a simulation and burst.fk_simulation_id is 8. Then I launch a copy or branch from that and that would have burst.fk_simulation_id 9. If I refresh after launching the copy or branch, in index the burst_config will have fk.simulation_id 8 so basically the simulation FROM WHICH we branched or copied will appear, not the one that we just launched, and in this case the buttons will appear...".

Now there is a task for this which I'm currently working on (TVB-2723).

In your question did you mean to put the line with is_simulator_copy in else?